modules/ROOT/pages/distributed-file-polling.adoc

```diff
@@ -0,0 +1,23 @@
+= Distributed File Polling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: file polling, cluster
+
+*_Enterprise Edition_*
+
+Some endpoints, such as the File Endpoint or FTP Endpoint, regularly poll directories and read certain files as they are created in the directories polled. These files may reside on a remote file system, including file systems of nodes belonging to a Mule High Availability (HA) Cluster.
+
+Up to Mule 3.3.x, all file poll requests sent to a cluster were channeled through a single node in that cluster. This ensured file consistency by avoiding the possibility of file system sync issues, but had the disadvantage of introducing a bottleneck, reducing the performance of the cluster.
+
+In the current version file polling for clusters' distributed file systems makes it possible to poll files in all cluster nodes. Enabled by default, this feature is used by the following endpoints:
```

Comment:
```suggestion
In the current version, file polling for clusters' distributed file systems makes it possible to poll files in all cluster nodes. Enabled by default, this feature is used by the following connectors:
```

Reply:
Explain the `primaryNodeOnly` feature of said connectors

Reply:
Hi Ale, Could you provide details on the `primaryNodeOnly` feature for the connectors? 
I was able to find this information https://github.com/mulesoft/mulesoft-docs/pull/2280/files regarding this field.
What you would like to be added for the connectors?

Reply:
Hi Ale, Could you provide details on the primaryNodeOnly feature for the connectors?
I was able to find this information https://github.com/mulesoft/mulesoft-docs/pull/2280/files regarding this field.
What you would like to be added for the connectors?

Reply:
It's exactly that. Just mention that connectors can be explicitly configured to only poll from the primary node, thus ignoring the default setting set by the runtime. This is only available in 4.x

Reply:
Updated.